bower.json

```diff
@@ -1,21 +1,21 @@
 {
   "name": "ember-cp-validations",
   "dependencies": {
-    "ember": "1.13.8",
+    "ember": "1.13.10",
     "ember-cli-shims": "0.0.6",
     "ember-cli-test-loader": "0.2.2",
     "ember-data": "~2.0.0",
     "ember-load-initializers": "0.1.7",
     "ember-qunit-notifications": "0.1.0",
     "ember-resolver": "~0.1.20",
-    "jquery": "^1.11.1",
+    "jquery": "~1.11.3",
```

Comment:
Pin to specific version for now. Releases of Ember will be out soon hopefully with the fix (it's already been fixed on master).